CefSharp.OffScreen/ChromiumWebBrowser.cs

```diff
@@ -373,11 +410,29 @@ protected virtual void Dispose(bool disposing)
         }
 
         /// <summary>
+        /// Gets the size of the popup.
+        /// </summary>
+        /// <param name="size">Size of the popup.</param>
+        public void GetPopupSize(ref Size size)
```

Comment:
It is probably not necessary since we don't want to be able to modify them outside the class. I'll change it to classic getters and see how it behaves.